playbook/app/pb_kits/playbook/pb_timestamp/timestamp.rb

```diff
@@ -73,12 +82,38 @@ def format_updated_string
 
       def format_elapsed_string
         user_string = show_user ? " by #{text}" : ""
-        datetime_string = " #{time_ago_in_words(pb_date_time.convert_to_timestamp)} ago"
+        datetime_string = " #{pb_time_ago(pb_date_time.convert_to_timestamp)} ago"
+        datetime_string[1] = hide_updated ? datetime_string[1].upcase : datetime_string[1]
         updated_string = hide_updated ? "" : "Last updated"
-
         "#{updated_string}#{user_string}#{datetime_string}"
       end
 
+      def pb_time_ago(value)
+        time_ago = DateTime.now.to_i - value.to_i
+        case time_ago
+        when (0...SECS_PER_MIN)
+          "a few seconds"
+        when (SECS_PER_MIN...SECS_PER_HOUR)
+          time = time_ago / SECS_PER_MIN
+          time == 1 ? "a minute" : "#{time_ago / SECS_PER_MIN} minutes"
+        when (SECS_PER_HOUR...SECS_PER_DAY)
+          time = time_ago / SECS_PER_HOUR
+          time == 1 ? "an hour" : "#{time_ago / SECS_PER_HOUR} hours"
+        when (SECS_PER_DAY...SECS_PER_WEEK)
+          time = time_ago / SECS_PER_DAY
+          time == 1 ? "a day" : "#{time_ago / SECS_PER_DAY} days"
+        when (SECS_PER_WEEK...SECS_PER_MONTH)
+          time = time_ago / SECS_PER_WEEK
+          time == 1 ? "a week" : "#{time_ago / SECS_PER_WEEK} weeks"
+        when (SECS_PER_MONTH...SECS_PER_YEAR)
+          time = time_ago / SECS_PER_MONTH
+          time == 1 ? "a month" : "#{time_ago / SECS_PER_MONTH} months"
+        when (SECS_PER_YEAR...SECS_PER_CENT)
+          time = time_ago / SECS_PER_YEAR
+          time == 1 ? "a year" : "#{time_ago / SECS_PER_YEAR} years"
```

Comment:
The React way is "320 days to 1+ year" (vs. 1+ year). 

Thinking a bit more, I kind of like the React way of doing things better. It'd be a bit weird to see a timestamp like "365 days ago" vs. "1 year ago".